docs/dev/annual_updates.rst

```diff
@@ -126,10 +126,13 @@ in :py:const:`pudl.extract.ferc714.TABLE_ENCODING` and that it may change over t
 
 A. EIA Forms
 ^^^^^^^^^^^^
-**3.A.1)** Use the Jupyter notebook ``devtools/eia-etl-debug.ipynb`` to run the extract
-process independently for each dataset. Given that there are hundreds of columns mapped
-across all the different EIA spreadsheets, you'll almost certainly find some typos or
-errors in the extract process and need to revise your work from step 2.
+**3.A.1)** Either attempt to materialize the raw assets for the EIA dataset you're
+working with in Dagster (learn more about Dagster in :doc:`run_the_etl`) or use the
+Jupyter notebook ``devtools/eia-etl-debug.ipynb`` to run the extract process
+independently for each dataset. Given that there are hundreds of columns mapped across
+all the different EIA spreadsheets, you'll almost certainly encounter typos or errors
+that will cause the extraction to fail. Interpret these errors and revise your work
+from step 2.
```

Comment:
"To speed up the debugging process, you can reload the code in Dagster once fixes have been made and materialize individual problematic assets"  - or something like this would be helpful here.